### PR TITLE
Use Scala 2.12.11

### PR DESCRIPTION
**Pull request by chbatey**

### CODE REVIEW

File: project/Dependencies.scala [hunk 1]
```diff
@@ -192,11 +192,7 @@ object Dependencies {
         Provided.activation // dockerClient needs javax.activation.DataSource in JDK 11+
       )
 
-  val actorTestkitTyped = l ++= Seq(
-        Provided.logback,
-        Provided.junit,
-        Provided.scalatest,
-        Test.scalatestJUnit)
+  val actorTestkitTyped = l ++= Seq(Provided.logback, Provided.junit, Provided.scalatest, Test.scalatestJUnit)
```
Comment (chbatey) [inline project/Dependencies.scala]: scalafmt

**Comment by raboof** [comment]
`sbt -Dakka.genjavadoc.enabled=true javaunidoc:doc` fails because https://repo1.maven.org/maven2/com/typesafe/genjavadoc/genjavadoc-plugin_2.12.11 is not there yet - https://github.com/lightbend/genjavadoc/pull/226 - I'll look into it.

**Review by raboof — APPROVED** [review]
genjavadoc released, so :+1: now AFAICS!

**Review by jrudolph — APPROVED** [review]
LGTM

**Pull request merged**